### DISCUSSION TITLE
Login to view asset info without username or password

### DISCUSSION THREAD

**Discussion by Flange0323** [snipe/snipe-it#13381] [Ideas]
Hi, 

Is there a feature that allows the ability to view an asset (after scanning the generated QR code) to allow someone to view the asset information & maintenance history and any associated files without having a login?

This would be useful for external audits. Obviously there wouldn't be any sensitive data on there. Just the list of when servicing was carried out.

Thanks

**Comment by TShiremanLCOG** [comment]
I don't think there is any way to do what you want without a login. Your best bet is probably to create an account, or get the email addresses of auditors to create their accounts. You would then assign those accounts to a limited access group that is only allow to see the information you described above. 

If you want a bad idea, you could write script that would auto-login someone to an auditor account with limited permissions. You'd then have to use custom QR codes to direct them to the auto-login page and then to the asset you want rather than using the generated codes.
